rest/data/host_test.go

```diff
@@ -209,6 +214,28 @@ func (s *HostConnectorSuite) TestFindByIdFail() {
 	s.Nil(h)
 }
 
+func (s *HostConnectorSuite) TestFindByIPFirst() {
```

Comment:
It would be a good idea to either combine FindHostByIPFirst and TestFindByIPLast, or use "ip5"  in TestFindByIPLast.

Reply:
combined